pkg/types/worker.go

```diff
@@ -25,8 +25,9 @@ type ContainerResourceUsage struct {
 }
 
 type Mount struct {
-	LocalPath string `json:"local_path"`
-	MountPath string `json:"mount_path"`
-	LinkPath  string `json:"link_path"`
-	ReadOnly  bool   `json:"read_only"`
+	LocalPath        string            `json:"local_path"`
+	MountPath        string            `json:"mount_path"`
+	LinkPath         string            `json:"link_path"`
+	ReadOnly         bool              `json:"read_only"`
+	MountPointConfig *MountPointConfig `json:"mountpoint_config"`
```

Comment:
it may make sense to make this into a pointer to an interface{}, and add an additional Type string, then you can use a switch(mount.type) and case it to this struct. To keep this extensible.